src/sas/sascalc/calculator/ausaxs/architecture.py

```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+
+class OS(Enum):
```

Comment:
Problem is that the `get_external_dependencies` script is run at build-time, before sasview is installed. So we cannot import modules from it yet. I thought about using relative imports, but decided against it since I wasn't sure if folders could potentially be shuffled around in the installed version.